src/testing/workspace-fixtures.ts

```diff
@@ -0,0 +1,58 @@
+import { defaultLocation } from 'src/analysis/utils/runtime-utils';
+import { locationTypes } from 'src/components/region-common';
+import { AzureWorkspace, GoogleWorkspace } from 'src/libs/workspace-utils';
+
+export const defaultAzureWorkspace: AzureWorkspace = {
+  workspace: {
+    authorizationDomain: [],
+    cloudPlatform: 'Azure',
+    name: 'test-azure-ws-name',
+    namespace: 'test-azure-ws-namespace',
+    workspaceId: 'fafbb550-62eb-4135-8b82-3ce4d53446af',
+    createdDate: '2023-02-15T19:17:15.711Z',
+    createdBy: 'justin@gmail.com',
+  },
+  azureContext: {
+    managedResourceGroupId: 'test-mrg',
+    subscriptionId: 'test-sub-id',
+    tenantId: 'test-tenant-id',
+  },
+  accessLevel: 'OWNER',
+  canShare: true,
+  canCompute: true,
+};
+
+export const defaultAzureStorageOptions = {
+  azureContainerRegion: 'eastus',
```

Comment:
Note that `eastus` will not be the default soon. Do we need this to be a specific value, or can it be a dummy value?